bench/common.mli

```diff
@@ -0,0 +1,21 @@
+module Make_context (Config : sig
+  val key_size : int
+
+  val hash_size : int
+
+  val value_size : int
+end) : sig
+  module Key : sig
+    include Index.Key with type t = string
+
+    val v : unit -> t
+  end
+
+  module Value : sig
+    include Index.Value with type t = string
+
+    val v : unit -> t
+  end
+end
+
+val with_timer : (unit -> 'a) -> 'a * float
```

Comment:
Potentially warrants a comment saying that the returned time is in units of seconds.